docs/scheduler.rst

```diff
@@ -32,161 +30,10 @@ Airflow production environment. To kick it off, all you need to do is
 execute ``airflow scheduler``. It will use the configuration specified in
 ``airflow.cfg``.
 
-Note that if you run a DAG on a ``schedule_interval`` of one day,
```

Comment:
This was quite an important point. I feel we should direct people to the new dag run page (where ever we decide it lives) from here too.

Reply:
This isn't done yet. A "see also" link is not strong enough.